AdobeStockImageAdminUi/view/adminhtml/web/js/action/saveLicensed.js

```diff
@@ -27,26 +27,25 @@ define([
             }).fail(function (message) {
                 deferred.reject(message);
             });
-
-        }
-
-        saveLicensedConfirmation(
-            pathUtility.generateImageName(title, id),
-            pathUtility.getImageExtension(contentType)
-        ).then(function (fileName) {
-            destinationPath = pathUtility.buildPath(directoryPath, fileName, contentType);
-            saveAction(
-                requestUrl,
-                id,
-                destinationPath
-            ).then(function () {
-                deferred.resolve(destinationPath);
-            }).fail(function (message) {
-                deferred.reject(message);
+        } else {
```

Comment:
I would personally prefer an early return rather than the else statement, but both are fine.

Reply:
Thanks, done!